pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -68,7 +68,8 @@ func fixHyperVInstalled() (bool, error) {
 	}
 
 	// We do need to error out as a restart might be needed (unfortunately no output redirect possible)
-	return true, errors.New("Please reboot your system")
+	logging.Error("Please reboot your system")
```

Comment:
Did you mean to stop the execution here? if so, this won't stop the execution, for that we need `logging.Fatal`.

Reply:
I wanted to print a message without stopping execution (since errors.New is printing the message and I did not want to lose that).

Reply:
@cfergeau why not stop? The rest of the execution can never succeed properly without this reboot

Reply:
`fixHyperVInstalled` does not need a forced reboot, as the user add can happen. But the user is only member of the group AFTER the reboot, so the `fixUserPartOfHyperVAdministrators` should stop.